src/shogun/evaluation/ROCEvaluation.cpp

```diff
@@ -19,8 +19,10 @@ CROCEvaluation::~CROCEvaluation()
 
 float64_t CROCEvaluation::evaluate(CLabels* predicted, CLabels* ground_truth)
 {
-	REQUIRE(predicted->get_label_type()==LT_BINARY, "ROC evalution requires binary labels.");
-	REQUIRE(ground_truth->get_label_type()==LT_BINARY, "ROC evalution requires binary labels.");
+	REQUIRE(predicted->get_label_type()==LT_BINARY, "Given predicted labels (%d) must be binary (%d).",
```

Comment:
great thanks!

One thing. If one of the provided pointers is NULL ,this segfaults.

`REQUIRE(predicted, "No predicted labels provided\n.")`

Reply:
Also, you are missing a `"\n"` at the end of the error msg